app/apollo/test/channel.fga.spec.js

```diff
@@ -167,499 +174,388 @@ describe('channel graphql test suite', () => {
-      }
+      console.error(JSON.stringify({'API response:': response && response.data ? response.data : 'unexpected response'}, null, 3));
+      console.error('Test failure, error: ', error);
       throw error;
     }
   });
 
   // channels fgaUser02
-  it('fgaUser02 has authentication to get ALLOWED channels', async () => {
+  it('fgaUser02 has authorization to get ALLOWED channels', async () => {
+    let response;
     try {
-      const result = await channelApi.channels(fgaToken02, {
+      response = await channelApi.channels(fgaToken02, {
         orgId: org01._id,
       });
-
-      expect(result.data.data.channels).to.have.length(1);
-
+      expect(response.data.data.channels).to.have.length(1);
+      expect(response.data.data.channels[0].name).to.equal(testChannel2.name);
     } catch (error) {
-      if (error.response) {
-        console.error('error encountered:  ', error.response.data);
-      } else {
-        console.error('error encountered:  ', error);
-      }
+      console.error(JSON.stringify({'API response:': response && response.data ? response.data : 'unexpected response'}, null, 3));
+      console.error('Test failure, error: ', error);
       throw error;
     }
   });
 
   // channel
-  it('fgaUser01 has authentication to get channel by channel uuid', async () => {
+  it('fgaUser01 has authorization to get channel 1 by channel uuid', async () => {
+    let response;
     try {
-      const {
-        data: {
-          data: { channel },
-        },
-      } = await channelApi.channel(fgaToken01, {
+      response = await channelApi.channel(fgaToken01, {
         orgId: org01._id,
-        uuid: 'testConfiguration1',
+        uuid: testChannel1.uuid,
       });
-
-      expect(channel.name).to.equal('test-configuration1');
-
+      expect(response.data.data.channel.uuid).to.equal(testChannel1.uuid);
     } catch (error) {
-      if (error.response) {
-        console.error('error encountered:  ', error.response.data);
-      } else {
-        console.error('error encountered:  ', error);
-      }
+      console.error(JSON.stringify({'API response:': response && response.data ? response.data : 'unexpected response'}, null, 3));
+      console.error('Test failure, error: ', error);
       throw error;
     }
   });
 
-  // channel without authentication
-  it('fgaUser01 does NOT have authentication to get channel by channel uuid', async () => {
+  // channel without authorization
+  it('fgaUser01 does NOT have authorization to get channel 2 by channel uuid', async () => {
+    let response;
     try {
-      const {
-        data: {
-          data: { channel },
-        },
-      } = await channelApi.channel(fgaToken01, {
+      response = await channelApi.channel(fgaToken01, {
         orgId: org01._id,
-        uuid: 'testConfiguration2',
+        uuid: testChannel2.uuid,
       });
-
-      expect(channel).to.equal(null);
-
+      console.log(response.data);
```

Comment:
extra log of response.data is unnecessary, probably dev leftover

Reply:
removed extra log

Reply:
resolved